Eevee.Sleep.Bot/Handlers/EventHandlers/GuildMemberUpdatedEventHandler.cs

```diff
@@ -86,6 +86,19 @@ IUser user
         );
     }
 
+    private static async Task HandleRolesAdded(ulong userId, ulong[] addedRoles) {
+        var roles = RoleRecordContoller.FindRoleIdsByUserId(userId);
+        var untrackedRole = TrackedRoleContoller
```

Comment:
Doesn't look about right to me, why would untracked roles getting role records? It should be only adding tracked roles to role records (that's why they're "tracked").

Reply:
This is obviously unnecessary...
`var roles = DiscordRoleRecordContoller.FindRoleIdsByUserId(userId);`

But I think we need to check tracked role.
This is because we want to add the newly added role to the user's ownership.
We can't add any role other than the one being tracked, so we exclude it here.

Reply:
> We can't add any role other than the one being tracked, so we exclude it here.
Shouldn't we check what roles are being tracked here instead?